test/methods.js

```diff
@@ -248,6 +250,39 @@ module.exports = {
         });
     },
 
+    "Reporter should be outputable to stdout and the file (#24)": function( test ) {
```

Comment:
Ah probably not. I wasn't really sure on conventions for naming tests since there is no documentation so I went off the test above it.